fuzz/fuzz_http3serverreq.cc

```diff
@@ -35,9 +45,23 @@ static int send_data(nghttp3_conn *conn) {
 extern "C" int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
   FuzzedDataProvider fuzzed_data_provider(data, size);
   nghttp3_callbacks callbacks{};
-  nghttp3_settings settings;
 
-  nghttp3_settings_default(&settings);
```

Comment:
This should be kept because this is how we initialize nghttp3_settings in practice.

Reply:
Makes sense. I made the change.